script/build_dgl.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+set -e
+
+usage() {
+cat << EOF
+usage: bash $0 OPTIONS
+examples:
+  Clean and restart a CPU only build: bash $0 -c
```

Comment:
Does $0 return `bash` if we execute
```
bash script/build_dgl.sh
```
?

Reply:
No, $0 returns the relative file path.
```
usage: bash script/build_dgl.sh OPTIONS
examples:
  Clean and restart a CPU only build: bash script/build_dgl.sh -c
  Clean and restart a CUDA build: bash script/build_dgl.sh -g
  Build incrementally: bash script/build_dgl.sh

Build DGL. By default, build incrementally on top of the current state.

OPTIONS:
  -h           Show this message.
  -c           Clean and restart CPU only build.
  -g           Clean and restart CUDA build.
```